src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -2074,6 +2077,15 @@ Widening and user defined conversions are not supported with generic types.</val
   </data>
   <data name="RecommendCaseInsensitiveStringComparerMessage" xml:space="preserve">
     <value>Prefer using 'StringComparer' to perform a case-insensitive comparison</value>
+  </data>
+    <data name="RecommendCaseInsensitiveStringEqualsTitle" xml:space="preserve">
+    <value>Prefer using 'string.Equals(string, StringComparison)' to perform case-insensitive string comparisons</value>
+  </data>
```

Comment:
Seems this is using the same ID as the existing string comparison analyzers, this could cause an issue or even if it is not cause an issue I believe only one of `RecommendCaseInsensitiveStringEqualsTitle` and `RecommendCaseInsensitiveStringComparisonTitle` will be used. It looks you can just use the same title `RecommendCaseInsensitiveStringComparisonTitle` for all.

Reply:
Fixed